openwisp_controller/config/base/device.py

```diff
@@ -83,6 +83,7 @@ class Meta:
             ('hardware_id', 'organization'),
         )
         abstract = True
+        verbose_name = app_settings.DEVICE_VERBOSE_NAME
 
```

Comment:
I don't think this change should be done in a model. Because each time you set a new value to the settings, you will need to run `./manage.py makemigrations` and `./manage.py migrate` and this won't be good at production. I suggest we look into other better solution like doing something in the `admin.py` or @nemesisdesign what do you say?

Reply:
@NoumbissiValere I think https://github.com/openwisp/openwisp-controller/pull/296#discussion_r499215099 will solve this issue. What are your thought?